modules/images/dominant-color/class-dominant-color-image-editor-gd.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @package performance-lab
+ * @group dominant-color
+ *
+ * @since n.e.x.t
+ */
+
+/**
+ * WordPress Image Editor Class for Image Manipulation through GD
+ * with dominant color detection
+ *
+ * @since n.e.x.t
+ *
+ * @see WP_Image_Editor
+ */
+class Dominant_Color_Image_Editor_GD extends WP_Image_Editor_GD {
+
+	/**
+	 * Get dominant color from a file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string|WP_Error Dominant hex color string.
+	 */
+	public function dominant_color_get_dominant_color() {
+
+		if ( ! $this->image ) {
+
```

Comment:
```suggestion
		if ( ! $this->image ) {
```

This space is needed. Let's remove this all in all places likes this.